tests/js/method/includes.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+// The base classes we are testing
+var express = require("../../../index.js");
```

Comment:
The first two and last two lines appear in each of these tests - should be factored into a common include - then you can move the two "real" includes simply into the test fixture file itself.